protocol/contracts/interfaces/v2/ITreasuryPriceIndexOracle.sol

```diff
@@ -12,10 +12,13 @@ import { ITempleElevatedAccess } from "contracts/interfaces/v2/access/ITempleEle
  */
 interface ITreasuryPriceIndexOracle is ITempleElevatedAccess {
     event TreasuryPriceIndexSet(uint96 oldTpi, uint96 newTpi);
```

Comment:
TODO: Remove unused events

Reply:
addressed in https://github.com/TempleDAO/temple/pull/1096/commits/a68b409070b6404c75eeae9ddc8a47516d6e6d1f